Scroller/EditableWrapper.js

```diff
@@ -74,7 +75,13 @@ const EditableWrapper = (props) => {
 		nextSpotlightRect: null,
 
 		// Flags
-		lastMoveDirection: null
+		lastMoveDirection: null,
+
+		// Last mouse position
+		mouseClientX: null,
```

Comment:
Let's rename this to `lastMouseClientX` or `lastMouseX`.

Reply:
It's fixed.